### ISSUE TITLE
Duplicating option field does not duplicate options

### ISSUE THREAD

**Issue by Notanotherdotcom** [ryancramerdesign/ProcessWire#1320]
This would be useful for lots of "yes/no" options, but then in such circumstances you would probably use a Page field instead if there's a lot of repetition. I just get the feeling that the expected action would be to duplicate the options too? Happy for folks to disagree with that of course.

Generally speaking I also think it should duplicate which templates a field was assigned to, or at least have the option under Duplicate/Clone This Field to do just that.

In fact, perhaps both of those suggestions would be useful to other fields and two checkboxes under the Duplicate/Clone This Field section would be the best option.

**Comment by LostKobrakai** [comment]
Slightly OT. I'd thought some time to copy FieldtypeCheckbox and make a "switch" by using a different inputfield. This could be done for yes/no fields as well.

**Comment by ryancramerdesign** [comment]
I agree it would make sense to duplicate the options too, but this one is a bit unique in that the options are stored off in a separate DB table, so there's a lot more to clone than just the field. It's possible to do, but something I'd like to revisit in the future.
